README.md

```diff
@@ -45,38 +47,59 @@ Additional checks for FASTQ file:
 - TXT
 - TSV
 
-## Run checkfiles in docker
 
-- Build the image
-
-`docker image build -f docker/Dockerfile -t checkfiles .`
-
-- There are some test file examples in file_examples folder. You can use one of them to run the build
+## Usage:
 
 ```bash
-docker run -it --privileged --platform linux/amd64 \
-    --env-file src/file_examples/bed_bed3+_env.txt \
-    -e AWS_ACCESS_KEY_ID=xxxxxxxx -e AWS_SECRET_ACCESS_KEY=xxxxxxxx\
-    -e ENCODE_ACCESS_KEY=xxxxxxxx -e ENCODE_SECRET_KEY=xxxxxxxx\
-    checkfiles
+$ python src/checkfiles/checkfiles.py -h
+usage: checkfiles.py [-h] [--uuid UUID] [--server SERVER] [--portal-key-id PORTAL_KEY_ID] [--portal-secret-key PORTAL_SECRET_KEY] [--patch] [--ignore-active-credentials]
+
+Checkfiles argumentparser
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --uuid UUID           UUID of the fileobject to be checked.
+  --server SERVER       igvf instance to check. https://api.sandbox.igvf.org for example
+  --portal-key-id PORTAL_KEY_ID
+                        Portal key id
+  --portal-secret-key PORTAL_SECRET_KEY
+                        Portal secret key
+  --patch               Patch the checked objects.
+  --ignore-active-credentials
+                        If this flag is set, then we omit checking if the file has unexpired upload credentials. There be dragons here, someone might change the underlying file after checking.
 ```
 
-## Testing
+Note that if `uuid` flag is not set, all the files with `upload_status=pending` will be checked. It may be prudent to use `screen` to run because the job can take several hours.
 
-Use the docker image to run test.
+## Running checkfiles on EC2
 
-```bash
-docker run -it --privileged --platform linux/amd64 \
-    -e AWS_ACCESS_KEY_ID=xxxxxxxx -e AWS_SECRET_ACCESS_KEY=xxxxxxxx\
-    -e ENCODE_ACCESS_KEY=xxxxxxxx -e ENCODE_SECRET_KEY=xxxxxxxx\
-    checkfiles pytest
-```
+In order to deploy a checkfiles EC2 instance (right now it makes most sense to run on igvf-staging account, to easily have access both to sandbox and prod from same machine).
+1. Log into igvf-staging aws console.
+2. Go to EC2 console and in the left tab select `AMIs`
+3. Select the newest `checkfiles__ami` (`ami-02001e7412122509a`)
+4. Click `Launch instance from AMI`.
+5. Select instsance type, and add storage. 200GB should be enough for anything.
```

Comment:
can you recommend the instance type here?

Reply:
Yea, I will add the recommendation to use the t2 instances.